pages/common/mongorestore.md

```diff
@@ -0,0 +1,19 @@
+# mongorestore
+
+> Utility to import a collection or database from a binary dump into a MongoDB instance.
+
+- Import database from folder containing a bson data dump:
```

Comment:
The language can be improved a bit. Its not very clear that the data is being imported "to" the database. I would love a "from" -> "to" sentence. How about "Import a bson data dump from a folder to a given database" ?